playground/examples/vue-bold/toolbar.vue

```diff
@@ -11,7 +11,7 @@ const editor = useEditor<EditorExtension>({ update: true })
   <div :class="Themes.TOOLBAR">
     <Button
       :pressed="editor.marks.bold.isActive()"
-      @click="() => editor.commands.toggleBold()"
+      @click="editor.commands.toggleBold()"
```

Comment:
Although `v-on` does also accept inline statements (i.e. without `() => `), and it's shorter, I'd like to still use function here. Since most other frameworks like React and Svelte only accept functions for event handler. So it would be easier to migrate examples between different frameworks if we all use functions.

Reply:
I suspected as much, which is why I separated the commits. You reverted, perfect !